lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -283,11 +283,10 @@
 .sort-dropdown {
     background-color: $color-bg-gray-blue-light;
     align-self: flex-end;
-    padding-bottom: 10px;
-    margin-left: 10px;
+    white-space: nowrap;
 
     p {
-        margin: 0 0 0 2px;
+        margin: 0;
```

Comment:
The placement of the dropdown arrow can be further improved with a negative margin, but this was not recommended in a previous PR. If preferred however for UI, I can add it.